packages/react/src/components/SuiteHeader/SuiteHeader.test.jsx

```diff
@@ -77,6 +78,16 @@ describe('SuiteHeader', () => {
     );
     expect(screen.getByRole('banner', { name: 'main header' })).toBeInTheDocument();
   });
+  it('renders with left sidenav toggle button (custom side nav support)', () => {
+    const mockOnSideNavToggled = jest.fn();
+    const wrapper = mount(
```

Comment:
All new test should be written using React testing library over enzyme.

Reply:
Sure, thanks for the heads up. I implemented with enzyme because I found an example of what I wanted to do in another unit test. But I'll convert the test to only use render/screen, instead of mount 👍